Makefile

```diff
@@ -60,14 +60,13 @@ endif
 ######################
 
 CAR_NONFREE_REMOTE ?= git@iis-git.ee.ethz.ch:carfield/carfield-nonfree.git
-CAR_NONFREE_COMMIT ?= a5351b2b161e896592b596492bff1abfac367aca
+CAR_NONFREE_COMMIT ?= 185ca77030bd9053b4a26c9cc0aa4d8d439d1cea
```

Comment:
Should we keep this or should we wait for it to be merged into main?

Reply:
It's merged into main but the commit hash is preserved, we can keep this imho